index.html

```diff
@@ -8,51 +8,58 @@
-                     some of my work and experience!<br> If you like what you see and have<br>
-                     a project you need coded,<br> 
-                    don’t hestiate to contact me.
-                </p>
-            </div>
-            <div class="social-media-class">
-                <ul class="social-media">
-                    <li class="social-media-links"><a href="#" class="LinkedIn-profile"><img src="./Images/Linkedin-icon.svg" alt="LinkedIn logo" class="linkedIn-class"></a></li>
-                    <li class="social-media-links"><a href="#" class="Twitter-profile"><img src="./Images/twitter.svg" alt="Twitter logo" class="Twitter-class"></a></li>
-                    <li class="social-media-links"><a href="#" class="Medium-profile"><img src="./Images/medium.svg" alt="Medium logo" class="Medium-class"></a></li>
-                    <li class="social-media-links"><a href="#" class="GitHub-profile"><img src="./Images/github.svg" alt="Github logo" class="Github-class"></a></li>
-                    <li class="social-media-links"><a href="#" class="Angellist-profile"><img src="./Images/angellist.svg" alt="Angellist logo" class="Angellist-class"></a></li>
-                </ul>
-            </div>
-        </div>
+        <section class="description-container">
+            <h1 class="description-title">
+                Hey there I'm<br>
+                Keenan<br>
+                <span>I'm a software<br>
+                developer</span>
+            </h1>
+            <p class="description-content">
+                I can help you build a product, <br>
+                    feature or website Look through<br>
+                    some of my work and experience!<br> 
+                    If you like what you see and have<br>
+                    a project you need coded,<br> 
+                don’t hestiate to contact me.
+            </p>
+            <ul class="social-media">
+                <li class="social-media-links"><a href="#" class="LinkedIn-profile"><img src="./Images/Linkedin-icon.svg" alt="LinkedIn logo" class="linkedIn-class"></a></li>
+                <li class="social-media-links"><a href="#" class="Twitter-profile"><img src="./Images/twitter.svg" alt="Twitter logo" class="Twitter-class"></a></li>
+                <li class="social-media-links"><a href="#" class="Medium-profile"><img src="./Images/medium.svg" alt="Medium logo" class="Medium-class"></a></li>
+                <li class="social-media-links"><a href="#" class="GitHub-profile"><img src="./Images/github.svg" alt="Github logo" class="Github-class"></a></li>
+                <li class="social-media-links"><a href="#" class="Angellist-profile"><img src="./Images/angellist.svg" alt="Angellist logo" class="Angellist-class"></a></li>
+            </ul>
+        </section>
         <!-- description container ends -->
       </div>
       <!-- Menu and about page start -->
-    <div class="menu-list">
+    <section class="menu-list">
```

Comment:
Kindly note these issues in the `Recent works` Section 🤗
- _The heading should be on one line_
- _Kindly expand the work cards to match the Figma design as shown below_

Your design:

![image](https://user-images.githubusercontent.com/73714615/187660934-22ad9528-7841-4c26-9246-5ec9219ddc6d.png)

Figma design:

![image](https://user-images.githubusercontent.com/73714615/187660979-f5cf3e84-5a5e-4c56-bec2-6e7b458151bb.png)